### PR TITLE
Revert expand collapse all buttons

### PR DESCRIPTION
**Pull request by itsyme**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [x] Others, please explain: Revert previous feature

**Overview of changes:**
Addresses the reverting in #2299.

**Anything you'd like to highlight/discuss:**

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**
Revert expand collapse all buttons

---

**Checklist:** :ballot_box_with_check:

- [x] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No unrelated changes

### CODE REVIEW

**Comment by itsyme** [comment]
> Hey @itsyme , just to check, this is purely a revert of the CollapseExpandAll PR (#2206) with no other changes?

yup!

**Review by tlylt — APPROVED** [review]
LGTM, thanks @itsyme

File: docs/userGuide/components/others.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 <div id="overview" class="lead">
 
-This page lists some other components that may be useful in creating education websites.
+This page lists some other components that may be useful in creating education websites. For now, there are only question and quiz components.
```
Comment (lhw-1) [inline docs/userGuide/components/others.md:21]: I know that we removed this sentence in #2206 so it is being added back, but I am not sure if this sentence is really necessary 😅

**Review by lhw-1 — APPROVED** [review]
Thanks for the PR @itsyme, LGTM!

**Pull request merged**